src/store/index.js

```diff
@@ -1,11 +1,26 @@
 import Vue from "vue";
 import Vuex from "vuex";
+import { getData } from "./actions";
+import { setLoadingState, setRestaurantsData } from "./mutations";
+import { getRestaurantData, getRestaurants, getStatus } from "./getters";
```

Comment:
I didn't question why did you split it into several files, that part is pretty obvious and not an overkill at all. My question was why you imported each exported function separately instead of importing them all as an object.